test/e2e/scale.go

```diff
@@ -73,11 +73,11 @@ func ScaleToWithin(t *testing.T, logger *logging.BaseLogger, scale int, duration
 		},
 	}
 	// These are the local (per-probe) and global (all probes) targets for the scale test.
-	// 95 = 19/20, so allow a single failure with the minimum number of probes, but expect
-	// us to have 3.5 9s overall.
+	// 90 = 18/20, so allow a single failure with the minimum number of probes, but expect
```

Comment:
Probably we need to change the comment to `to allow 2 failures`?
Would be also good to leave a todo with a reminder referring to istio 1.1 migration.

Reply:
Done